src/TodoListItem.js

```diff
@@ -2,21 +2,17 @@ import React from "react";
 
 function TodoListItem({ todo: { title, id }, onRemoveTodo }) {
   return (
-    <div>
+    <ul> 
```

Comment:
Requires changes:
A parent `ul` is defined within the [TodoList](https://github.com/AbebeTesso/flamingo-react-todo/blob/5856f7dafae065effd8ff704d079ad3cd724f748/src/TodoList.js#L7) component, therefore, this `ul` element within `TodoListItem` needs to be removed. Otherwise, your HTML structure will be...
```html
<ul>
  <ul>
    <li>todo</li>
  </ul>
  <ul>
    <li>todo</li>
  </ul>
  <ul>
    <li>todo</li>
  </ul>
   ...etc
</ul>
```
When it should be:
```html
<ul>
  <li>todo</li>
  <li>todo</li>
  <li>todo</li>
   ...etc
</ul>
```